etcdmain/help.go

```diff
@@ -172,7 +172,11 @@ profiling flags:
 	--enable-pprof 'false'
 		Enable runtime profiling data via HTTP server. Address is at client URL + "/debug/pprof/"
 	--metrics 'basic'
-	  Set level of detail for exported metrics, specify 'extensive' to include histogram metrics.
+		Set level of detail for exported metrics, specify 'extensive' to include histogram metrics.
+	--expose-metrics 'false'
+		Additionally expose '/metrics' endpoints via '/metrics-urls'.
+	--metrics-urls 'http://localhost:9379'
```

Comment:
`--listen-metrics-urls` and default to "" so `expose-metrics` isn't needed? blank list => no listen